File: articles/distribution-shift-and-model-decay/java/DecayRisk.java
public class DecayRisk {
    public static void main(String[] args) {
        double inputDrift = 0.31;
        double labelDrift = 0.16;
        double performanceDecay = 0.10;
        double calibrationGap = 0.14;
        double subgroupGap = 0.15;
        double overrideRate = 0.11;
        double score = (inputDrift + labelDrift + performanceDecay + calibrationGap + subgroupGap + overrideRate) / 6.0;
        System.out.printf("decay_risk_score=%.4f%n", score);
    }
}

File: articles/distribution-shift-and-model-decay/fortran/decay_risk.f90
program decay_risk
  implicit none
  real :: input_drift, label_drift, performance_decay, calibration_gap, subgroup_gap, override_rate, score
  input_drift = 0.31
  label_drift = 0.16
  performance_decay = 0.10
  calibration_gap = 0.14
  subgroup_gap = 0.15
  override_rate = 0.11
  score = (input_drift + label_drift + performance_decay + calibration_gap + subgroup_gap + override_rate) / 6.0
  print *, "decay_risk_score=", score
end program decay_risk
